### PR TITLE
Skip building Samples.OracleMDA.* projects

### PR DESCRIPTION
**Pull request by lucaspimentel**
Remove `Samples.OracleMDA` and `Samples.OracleMDA.Core` from build scripts. We don't use them and they are causing issues with the build in #1468. We still need to figure out how to run them in CI anyway, and hopefully by that point we'll have build scripts that are easier to work with (see #1476).

Notes: 
- I added PR comments to the two changes I made manually. Everything else it auto-generated by the `GeneratePackageVersions` tool.
- I canceled some of the pipelines which were not necessary (benchmarks, Crank, runner). Let me know if you want me to run them before merging.

### CODE REVIEW

File: build/PackageVersionsGeneratorDefinitions.json [hunk 1]
```diff
@@ -78,19 +78,5 @@
...
-    "NugetPackageSearchName": "Oracle.ManagedDataAccess",
-    "MinVersion": "18.3.0",
-    "MaxVersionExclusive": "20.0.0"
-  },
-  {
-    "IntegrationName": "OracleMDACore",
-    "SampleProjectName": "Samples.OracleMDA.Core",
-    "NugetPackageSearchName": "Oracle.ManagedDataAccess.Core",
-    "MinVersion": "2.18.3",
-    "MaxVersionExclusive": "3.0.0"
```
Comment (lucaspimentel) [inline build/PackageVersionsGeneratorDefinitions.json:94]: Manual change

File: build/docker/build.sh [hunk 2]
```diff
@@ -42,7 +42,7 @@ fi
 
 dotnet publish -f $publishTargetFramework -c $buildConfiguration test/test-applications/instrumentation/CallTargetNativeTest/CallTargetNativeTest.csproj -p:Configuration=$buildConfiguration -p:ManagedProfilerOutputDirectory="$PUBLISH_OUTPUT"
 
-for sample in Samples.Elasticsearch Samples.Elasticsearch.V5 Samples.ServiceStack.Redis Samples.StackExchange.Redis Samples.SqlServer Samples.Microsoft.Data.SqlClient Samples.MongoDB Samples.HttpMessageHandler Samples.WebRequest Samples.Npgsql Samples.MySql Samples.GraphQL Samples.Dapper Samples.NoMultiLoader Samples.RabbitMQ Samples.RuntimeMetrics Samples.FakeDbCommand Samples.Microsoft.Data.Sqlite Samples.SQLite.Core Samples.OracleMDA Samples.OracleMDA.Core ; do
```
Comment (lucaspimentel) [inline build/docker/build.sh:45]: Manual change

**Comment by andrewlock** [comment]
I think this is fine if it unblocks you, but also, it's pretty bad we're not running this in CI. AFACIT [we can run Oracle in Docker](https://oracle-base.com/articles/linux/docker-oracle-database-on-docker), so is it just a licensing issue? If so, is it worth looking into testing directly against their cloud, e.g. [using their free tier](https://www.oracle.com/uk/cloud/free/). Even if we only ran it nightly (to avoid outages or flakiness blocking CI etc)?

**Review by tonyredondo — APPROVED** [review]

**Review by zacharycmontoya — APPROVED** [review]
LGTM. This is worth creating an internal ticket so we are aware of this testing gap

**Pull request merged**